packages/client/components/StageTimerDisplayGauge.tsx

```diff
@@ -1,5 +1,6 @@
 import styled from '@emotion/styled'
-import React from 'react'
+import React, {useEffect, useRef} from 'react'
+import sound from '../../../static/sounds/tic-tac.mp3'
```

Comment:
+1 I like the sound although it's not super obvious to me that it signals the end of a timer. Perhaps the countdown game show sound would be more obvious? 

Also, very nit-picky, but we could change the variable name from `sound` to `endTimerSound`

Reply:
I like the sound as it's not very distracting. I would love to try it for some time to get a feel how it behaves in a real meeting. My first reaction was to put it behind a feature flag, but I don't think it's unsafe to try it as is.
@ackernaut As this is introducing sound into our app, I think you might want to check it.